src/main/kotlin/com/terraformation/backend/documentproducer/db/variable/AllVariableCsvVariable.kt

```diff
@@ -47,13 +47,15 @@ data class AllVariableCsvVariable(
     val deliverableId: DeliverableId?,
     /** Column 15/O - Deliverable Question */
     val deliverableQuestion: String?,
-    /** Column 16/P - Dependency - Variable Stable ID */
+    /** Column 16/P - Deliverable Question Position */
+    val deliverableQuestionPosition: Int?,
```

Comment:
Should this be an explicit value in the sheet, or should we calculate it based on the order of the questions in the sheet? Calculating it means no possibility of things like duplicate or missing position numbers, and I suspect "questions are in the same order they appear in the spreadsheet" will be easier for admins to use, especially if they're inserting questions toward the beginning of an existing deliverable (in which case they'd have to manually renumber the later ones).

Reply:
Good call, I misunderstood the slack thread about this. I closed this PR out and did it the right way in https://github.com/terraware/terraware-server/pull/2090. 

I also went with `deliverable_position` instead of `deliverable_question_position` because it is not related to the `deliverable_question` field.